controllers/topology/cluster_controller.go

```diff
@@ -108,6 +110,7 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_
 		// Error reading the object - requeue the request.
 		return ctrl.Result{}, err
 	}
+	cluster.Kind = "Cluster"
```

Comment:
curious why is this needed?

Reply:
see https://github.com/kubernetes-sigs/cluster-api/pull/5631#discussion_r746945426

Reply:
Could we actually use the scheme here to set the GVK?